__tests__/app.test.js

```diff
@@ -12,15 +12,96 @@ const app = require("../index");
 beforeEach(() => seed({ topicData, articleData, userData, commentData }));
 
 afterAll(() => db.end());
+describe("Error tests", () => {
+  describe("testing 404 endpoint when given non existing URL", () => {
+    test("should output 404 and a message when givien a invalid url", () => {
```

Comment:
test("should return output 404 and a message when invalid url is entered", () => {